strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImpl.java

```diff
@@ -160,6 +160,11 @@ public void setPort(int port) throws IOException
         modifyInLock(configuration -> configuration.setPort(port));
     }
 
+    @Override
+    public void setDigestAlgorithmSet(Set<String> digestAlgorithmSet) throws IOException {
+        modifyInLock(configuration -> configuration.setDigestAlgorithmSet(digestAlgorithmSet));
+    }
```

Comment:
Cosmetic: Braces on new lines, please.

```suggestion
    public void setDigestAlgorithmSet(Set<String> digestAlgorithmSet)
            throws IOException
    {
        modifyInLock(configuration -> configuration.setDigestAlgorithmSet(digestAlgorithmSet));
    }
```